scripts/operator/1_deploy.rootchain.sh

```diff
@@ -20,6 +20,12 @@ echo "Deploy rootchain contract and others"
     --unlock $OPERATOR \
     --password "pwd.pass" \
     --rootchain.sender $OPERATOR \
+    --rootchain.deploygasprice $DEPLOY_GASPRICE \
+    --rootchain.gasprice $COMMIT_GASPRICE \
```

Comment:
Two command, deploygasprice and gasprice, can not work together. Is this command work?